app/src/main/java/com/alphawallet/app/service/TokensService.java

```diff
@@ -112,7 +112,7 @@ private void checkUnknownTokens()
 
```

Comment:
TokensService should not know about 'isEnabled', it is a backend service and should be 100% decoupled from UI. This need to be refactored. See above comment about 'isEnabled' not being in the ContractLocator class.

Reply:
removed